gammapy/modeling/parameter.py

```diff
@@ -843,6 +846,26 @@ def __exit__(self, type, value, traceback):
 
 
 class PriorParameter(Parameter):
+    """Parameter of a `~gammapy.modeling.models.Prior`.
+
+    A prior is a probability density function of a model parameter and can take different forms, including but not limited to Gaussian
+    distributions and uniform distributions. The prior includes information or knowledge about the dataset or the
+    parameters of the fit.
+
+    Examples
+    --------
+    For a usage example see :doc:`/tutorials/api/priors` tutorial.
+
+     Parameters
+     ----------
+     name : str
+         Name.
+     value : float or `~astropy.units.Quantity`
+         Value.
+     unit : `~astropy.units.Unit` or str, optional
+         Unit. Default is "".
```

Comment:
```suggestion
    Examples
    --------
    For a usage example see :doc:`/tutorials/api/priors` tutorial.

    Parameters
    ----------
    name : str
        Name.
    value : float or `~astropy.units.Quantity`
        Value.
    unit : `~astropy.units.Unit` or str, optional
        Unit. Default is "".
```
I think there is a misalignment.